### PR TITLE
Add tests for conformance of pyvista colors to the web color standard

### PR DESCRIPTION
**Pull request by user27182**
### Overview

Follow-up to #6592. 

- Make sure colors match the web standard: https://www.w3.org/TR/css-color-4/#named-colors 
- `brown` was found to be non-conformant and is modified by this PR.
- Tests for color synonyms and color uniqueness are also added.

EDIT: These are the same colors referred to as CSS4 in Matplotlib:
https://matplotlib.org/stable/gallery/color/named_colors.html#css-colors

### CODE REVIEW

File: tests/plotting/test_colors.py [hunk 1]
```diff
@@ -121,3 +122,187 @@ def test_color():
 def test_color_opacity():
     color = pv.Color(opacity=0.5)
     assert color.opacity == 128
+
+
+# Define standardized names and hex values of web colors defined here:
+# https://www.w3.org/TR/css-color-4/#named-colors
+WEB_COLORS: final = {
+    'aliceblue': '#F0F8FF',
```
Comment (tkoyama010) [inline tests/plotting/test_colors.py:130]: It's hard to check if these color codes are correct, so it's a good idea to use the Matplotlib function. However, we need to check that Matplotlib covers all of the codes here.
```python
>>> from matplotlib import colors
>>> print(colors.cnames['aliceblue'])
#F0F8FF
```

**Review by tkoyama010 — CHANGES_REQUESTED** [review]

File: tests/plotting/test_colors.py (hunk 1 above)
Comment (user27182) [reply tests/plotting/test_colors.py]: Fixed in https://github.com/pyvista/pyvista/pull/6593/commits/fe593cf7249433a96b275ba5fc3be392fbb6f98e

File: tests/plotting/test_colors.py [hunk 2]
```diff
@@ -121,3 +122,33 @@ def test_color():
...
+
+
+def test_css4_colors(css4_color):
+    name, value = css4_color
+    assert pv.Color(name).hex_rgb.lower() == value.lower()
+
+
+def test_color_synonyms(color_synonym):
+    color = pv.Color(color_synonym)
+    assert color.name
```
Comment (tkoyama010) [inline tests/plotting/test_colors.py:148]:
```suggestion
    assert isinstance(color, pv.Color)
```

**Review by tkoyama010 — CHANGES_REQUESTED** [review]

**Review by tkoyama010 — APPROVED** [review]
LGTM

**Review by tkoyama010 — APPROVED** [review]

**Pull request merged**